contents/handbook/people/hiring-process.md

```diff
@@ -14,34 +14,103 @@ At the moment, we're not too worried about having a clearly defined job spec. Po
+
+The next section is an example agenda. Make sure to insert the appropriate stage and links for the role. Keep the diversity message in as part of being inclusive to all candidates.
+
+#### Example agenda
+
+Hey Mary,
+
+Welcome to the first step of our hiring process. We're excited to meet you!
+
+This is a cultural discussion, and it's to understand if we'd be setting each other for success by working together.
+
+Here is [a little info about the role](https://posthog.com/careers/designer).
+
+We are a transparent by default company since [we value being open source](https://posthog.com/handbook/company/values) - you can read our entire [handbook online](https://posthog.com/handbook). Here's a bit about [working at PostHog](https://posthog.com/careers). 
+
+We're proud to be a group of [misfits](https://posthog.com/handbook/company/diversity/). Diversity in our hiring matters greatly to us.
+
+### Outreach
+
+Outreach has a few advantages:
+
+* We can approach people with very specific experience
+* It allows us to encourage candidates from a wider range of backgrounds to apply 
+
+It is possible to research a list of potential candidates through:
+
+* Twitter
+* LinkedIn
+* Behance
+* Dribble
+
+It is important before starting outreach like this that you consider *why* a candidate messaged like this may move to us so your note to them can explain why you felt it might be a nice fit.
+
+### YC's Work at a Startup
+
+Since PostHog is a YC company, we can place job ads in YC's [Work at a Startup list](https://www.workatastartup.com/jobs).
+
+### Referrals
+
+* From our own team
+* From our investors
+* From the PostHog community (the users Slack Group, and posting on our social media profiles for our followers to see)
+* From the YC community (Slack/Whatsapp/forum)
+
+### Job boards
+
+#### Design
+
+We are learning which boards work well:
+
+- [Behance](https://www.behance.net/adobetalent)
+- [Dribbble](https://dribbble.com/jobs/new)
+
+#### Engineering
 
 - HackerNews Who's Hiring
     - Tend to get high quality candidates, and people interested in working at startups. 
     - See [Tim's comment history](https://news.ycombinator.com/threads?id=timgl) for a template.
-- RemoteOK
-    - High volume of candidates, but much lower quality.
-- AngelList
+- [Tech Ladies](https://www.hiretechladies.com/)
+    - Paid, but helps us achieve a more diverse group of candidates.
+- [AngelList](https://angel.co)
     - We found Eric through there. Higher quality than RemoteOK and pretty high volume.
+- [RemoteOK](https://remoteok.io/)
+    - High volume of candidates, but much lower quality.
 
 ## Interviewing
 
 The stages of interviewing are:
 
 - Application
-- Interview with Tim
-- Interview with James
-- Interview with someone from the team
-- 3 days of contracting work
+   - decide if we will interview
+- Interview with the hiring team (this will always be James, Tim and 1 or more people that would work closely with the candidate day-to-day).
```

Comment:
I think this should be:
- First interview with James, go/no-go decision
- 1 hour block, interview with Tim and 1 other person that would work closely with the candidate day-to-day

That way we can speed up the process a bit

Reply:
Unclear: do you mean one 1-hour block with two people, or 2 separate 1-hour blocks?

Reply:
I agree with doing go/no-go after 1 interview each time, that'll stop me putting through maybe candidates who probably will have a 95% failure rate later.

Re the 1 hour block - I read your intention as two people interviewing over the course of 1 hour at the same time as each other. Regardless of what you meant (!), I think this *could* work well... to the point where I'd be in favour of this.

I think these pros outweigh the cons:

# pros

* when you are interviewing, you are having to form questions and listen at the same time. Even if you have prepared some stuff in advance. I think 2 people could raise the quality of the interview. 
* it'd also enable us to improve our own ability to do interviews.
* saves the candidate 1 interview of time
* it would make the interview more challenging, which would help create the first impressions that this is a challenging place to work, which we want to set as a standard
* could enable faster decisions which reduces chance of losing candidate if they're being hired by multiple places=

# cons

* we'd miss some breadth of questions I suspect in exchange for going deeper. We would need to be a little more careful with having a plan for the interview so we don't miss any major areas, but I think going deeper could get us past the veneer it's easy to put on for surface level questions
* it would increase the pressure of the interview. This is less inclusive for candidates who are bad at interviews but great at the actual work.
* a little annoying to schedule

Reply:
Sorry my suggestion was, make sure we have the candidate for an hour, then me and one other person do a half hour interview each. This way you avoid having to wait 3-4 days between me interviewing and scheduling the 3rd interview

Reply:
Regardless of your intention, I still think the 2 at the same idea is worth trying for the reasons above.

Do you disagree? Open to pushback.

Reply:
Yeah def worth trying